plugins/GSdx/Renderers/DX11/GSRendererDX11.cpp

```diff
@@ -501,6 +501,114 @@ void GSRendererDX11::EmulateChannelShuffle(GSTexture** rt, const GSTextureCache:
 	}
 }
 
+void GSRendererDX11::EmulateBlending()
+{
+	// Partial port of OGL SW blending. Currently only works for accumulation blend or when
+	// m_sw_blending >= ACC_BLEND_HIGH_D3D11 since it has not been fully tested. There is no check
+	// currently for primitive overlap like in OGL but it should give better results for post-processing effects
+	// that are done by non-overlapping sprites.
+	const GIFRegALPHA& ALPHA = m_context->ALPHA;
+	bool sw_blending         = false;
+
+	m_om_bsel.abe = PRIM->ABE || (PRIM->AA1 && m_vt.m_primclass == GS_LINE_CLASS);
+	m_om_bsel.a = ALPHA.A;
+	m_om_bsel.b = ALPHA.B;
+	m_om_bsel.c = ALPHA.C;
+	m_om_bsel.d = ALPHA.D;
+
+	if (m_env.PABE.PABE)
+	{
+		if (m_om_bsel.a == 0 && m_om_bsel.b == 1 && m_om_bsel.c == 0 && m_om_bsel.d == 1)
+		{
+			// this works because with PABE alpha blending is on when alpha >= 0x80, but since the pixel shader
+			// cannot output anything over 0x80 (== 1.0) blending with 0x80 or turning it off gives the same result
+
+			m_om_bsel.abe = 0;
+		}
+		else
+		{
+			//Breath of Fire Dragon Quarter triggers this in battles. Graphics are fine though.
+			//ASSERT(0);
+		}
+	}
+
+	// Accumulation blend
+	uint8 blend_index  = uint8(((ALPHA.A * 3 + ALPHA.B) * 3 + ALPHA.C) * 3 + ALPHA.D);
+	int blend_flag = m_dev->GetBlendFlags(blend_index);
```

Comment:
Maybe use dev instead of m_dev here too, for consistency.

Reply:
So he can have a useless cast?
`GetBlendFlags` is a GSDevice function there's so reason to call it on `GSDevice11`

Reply:
@lightningterror  True, it would be more consistent with the OGL EmulateBlending(). However it is not necessary to use dev = (GSDevice11*)m_dev here because the blending setting are handled using the OMBlendSelector struct (m_om_bssel) instead of through GSDevice11 methods.